Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -44,7 +44,7 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         this.delegate = delegate;
 
         this._cleared = true;
-        this._previousMessageView = null;
+        this._lastConseoleMessageViewForTargetMap = new WeakMap();
```

Comment:
typo in the name "Conseole" (and also there's no need for the "Map" since the "fooForBar" pattern implies that it's a `Map`)

Style: we dont normally add `()` when there's no arguments with `new`

```js
this._lastConsoleMessageViewForTarget = new WeakMap;
```